charts/prometheus-adapter/templates/custom-metrics-apiservice.yaml

```diff
@@ -6,16 +6,17 @@ apiVersion: apiregistration.k8s.io/v1beta1
 {{- end }}
 kind: APIService
 metadata:
-{{- if .Values.certManager.enabled }}
+{{- if or .Values.certManager.enabled .Values.customAnnotations }}
   annotations:
     certmanager.k8s.io/inject-ca-from: {{ printf "%s/%s-root-cert" .Release.Namespace (include "k8s-prometheus-adapter.fullname" .) | quote }}
     cert-manager.io/inject-ca-from: {{ printf "%s/%s-root-cert" .Release.Namespace (include "k8s-prometheus-adapter.fullname" .) | quote }}
+    {{- if .Values.customAnnotations }}
+    {{- toYaml .Values.customAnnotations | nindent 4 }}
+    {{- end }}
 {{- end }}
   labels:
     app: {{ template "k8s-prometheus-adapter.name" . }}
```

Comment:
this is redundant with: "app.kubernetes.io/part-of"

Reply:
done